docs/project-charter.md

```diff
@@ -0,0 +1,196 @@
+_document also connects a project's community leadership and governance with the_
+_OpenJS Foundation's governance._
+
+## Section 0: Guiding Principles (optional)
+
+_directions: provide a concise, high-level statement about_
+_the project's long-term principles, values, or mission._
+
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-1-guiding-principle)
+
+LoopBack (herein "Project") is part of the OpenJS Foundation which operates
+transparently, openly, collaboratively and ethically. Project proposals,
+timelines, and statuses must be open and visible to outsiders.
+
+## Section 1: Scope
+
+_directions: Include a 3-4 sentence summary of what the project does,_
+_and/or what problems it solves. Imagine trying to explain your work_
+_to a colleague who is familiar with related technical concepts but unfamiliar_
+_with the project. You may also want to describe the project's value to community_
+_and/or business stakeholders._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#scope)
+
+The Project intends to create and maintain a collection of Node.js packages to
+provide developers with the tools to create enterprise solutions which require
+interaction between a diverse set of APIs. Notably, the Project emphasizes the
+creation of extensible, open APIs to empower developers with flexibility and
+modularity. This allows the packages easily adapted to a wide range of
+use-cases and allows different subsets of the Project to be used as needed.
+
+### 1.1: In-scope (optional)
+
+_directions: list or bullet out problem spaces, use cases, repositories_
+_or other projects which are included with the work but may not be readily_
+_apparent. This may help differentiate the project from other solutions in the_
+_space. If you are not using this section, please indicate your intent with the_
+_phrase, 'Section Intentionally Left Blank'._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#in-scope)
+
+- Maintenance of existing Node.js packages
+  - Supporting latest versions of Node.js
+  - Managing deprecation
+- Modernisation and extension of existing Node.js packages
+- Creating new Node.js packages
+- Documenting and standardising interfaces for common interactions
+
+### 1.2: Out-of-Scope (optional)
+
+_directions: list or bullet out areas that may be seen to be related but are_
+_not included in the scope of this project. This may help clarify the kind of_
+_features, contributions, issues or problems the project is looking for._
+_If you are not using this section, please indicate your intent with the_
+_phrase, 'Section Intentionally Left Blank'._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#out-of-scope)
+
+Section Intentionally Left Blank
```

Comment:
I also looked at the one from fastify. Might be good to use it as a reference: https://github.com/fastify/fastify/blob/main/PROJECT_CHARTER.md#12-out-of-scope.

particulary:
- Support versions of Node.js at EOL (end of life) stage
- Contributions that violates the Code of Conduct